packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -58,7 +58,7 @@ const addIsActive =
 
 export const AppLayoutSidebarNav = ({
 	activePath,
-	background,
```

Comment:
This shouldn't be necessary since `AppLayoutSidebar` has `bodyAlt` as its default